types/api/query/chain.go

```diff
@@ -15,6 +15,15 @@ type ResponseByUrl struct {
 	state.Object
 }
 
+type RequestDirectory struct {
+	RequestByUrl
+	ExpandChains types.Bool
+}
+
+type ResponseDirectory struct {
+	state.Object
+}
```

Comment:
This type is not used anywhere, so it should be removed. I don't know why `ResponseByUrl` exists, we should delete that too (not necessarily in this PR).

Reply:
Done (in next commit)
I also deleted ResponseByUrl since my cursor was already there...